### PR TITLE
Gperf

### PR DESCRIPTION
**Pull request by convolvatron**
cleanups and bug fixes - currently going over it myself, will add comment when I'm finished

### CODE REVIEW

**Comment by convolvatron** [comment]
did a first whitespace pass...should maybe leave out the svg stuff?

File: nfsv4/test/svg.c [hunk 1]
```diff
@@ -0,0 +1,140 @@
+#include <runtime.h>
```
Comment (jssmith) [inline nfsv4/test/svg.c]: Please confirm that we will be able to release this code under an open source license (probably MIT).

Comment (convolvatron) [reply nfsv4/test/svg.c]: yes, this is owned by me, but (a), its not complete and (b), i'm quite happy to have it released under mit

File: nfsv4/xdr.c [hunk 2]
```diff
@@ -94,6 +114,77 @@ void push_claim_deleg_cur(buffer b, buffer name)
...
+    read_beu32(f->c, b); // atomic
+    u32 before = read_beu64(f->c, b); // before
+    u32 after = read_beu64(f->c, b); // after
+    read_beu32(f->c, b); // rflags
+    read_beu32(f->c, b); // bitmap4 attr
+    read_beu32(f->c, b); // xxx - no -idea
+    read_beu32(f->c, b); // xxx - no -idea
+    u32 delegation_type = read_beu32(f->c, b);
+
+    switch (delegation_type) {
```
[the remark this answers is not in the document]
Comment (convolvatron) [reply nfsv4/xdr.c]: yeah...i put it in even though we weren't using it, and apparently never will. the general desire here though is to fully deconstruct the frames. if you dont keep reading along properly, any later contents will be nonsense. so I think its ok to leave bits of the base level frame parser even thought we dont expect to meaningfully use them

**Pull request merged**